test/test.js

```diff
@@ -2634,6 +2655,15 @@ describe('Request', function () {
 	});
 });
 
+describe('Body', function () {
+	it('should return null for a stream\'s total bytes, ignoring size', () => {
+		const fetchBody = new Body(fs.createReadStream(path.join(__dirname, 'dummy.txt')), {
+			size: 2048
```

Comment:
What do you think about throwing an error if `size` is set when `Body` is constructed with a stream?

If the library previously just ignored `size` (can't tell), then I guess we might as well continue ignoring it, just wondering if `size` is at all useful here and if ignoring it might mask any other errors.

Reply:
If we did enforce such an error I actually might suggest doing so at the level of `fetch`, i.e. when the `body` and `size` options conflict.

Reply:
The library previously ignored size so I'd switch back to that as a patch and a more opinionated API design could be implemented later. 

Based on my reading of the spec, it doesn't appear that [`Content-Length` is necessary](https://fetch.spec.whatwg.org/#concept-read-all-bytes-from-readablestream) when reading from a stream. Additionally, the http/https request functions default to `Transfer-Encoding: chunked` which doesn't require a `Content-Length`.

Reply:
For this particular test, if the purpose is to test `getTotalBytes` then I think a new `Body` section isn't needed, you can put it under `node-fetch` section.